collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -954,6 +1010,24 @@ def query_factory(name, version=NO_VERSION):
             ['brin_summarize', 'brin summarize', 'absolute']
         ]
     },
+    'emergency_autovacuum': {
+        'options': [None, 'Percent towards emergency autovac', 'percent', 'autovacuum', 'postgres.emergency_autovacuum', 'line'],
+        'lines': [
+            ['percent_towards_emergency_autovac', 'percent', 'absolute']
+        ]
+    },
+    'tx_wraparound_oldest_current_xid': {
+        'options': [None, 'Oldest current XID', 'xid', 'tx_wraparound', 'postgres.tx_wraparound_oldest_current_xid', 'line'],
```

Comment:
What is `xid`? Some id? Why we need it as a metric?

Reply:
xid means "transaction id"

Reply:
Thanks. 2nd question remains. Why are we interested in it as a metric? @mjtice

Reply:
This could be a good one to both trend over time to determine change rate.  Additionally alarms could be set on this as another warning for approaching the max xid allowed by postgres before transaction wraparound happens.